lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8437,6 +8438,107 @@ ClangImporter::Implementation::importSwiftAttrAttributes(Decl *MappedDecl) {
   }
 }
 
+namespace {
+class PointerParamInfo {
+public:
+  virtual void print(clang::ASTContext &ctx, llvm::raw_ostream &out) const = 0;
+  virtual ~PointerParamInfo() {}
```

Comment:
Nit: I think usually `= default;` is preferred over `{}`. Although I do not expect a difference in this particular case.